### ISSUE TITLE
Add a magnifying glass icon to the main screen of the application to search for POI

### ISSUE THREAD

**Issue by Korb** [Zverik/every_door#671]
In the case of a very dense POI location, it is easier to search using keywords by selecting the desired location from the list of found ones, sorted by distance from the center of the screen.  Currently, the location search functionality is not found.  Installed version:

4.0.0 (373) Suggested
Added on 10/10/2023
Repository: F-Droid
Size: 26.3 MiB
Android: 4.4+
Requires: arm64-v8a

**Comment by Zverik** [comment]
Duplicate of #24. I understand the intention, but maybe it should be done a different way. Looking for every single POI is time-consuming.
